server/routes/aboutPerson/locale.json

```diff
@@ -14,9 +14,9 @@
       "lastUpdated": "Last updated on {{ lastUpdatedDate }}.",
       "RoSH": "RoSH (Risk of Serious Harm)",
       "riskOfReoffending": "Risk of reoffending",
-      "linkedToRoSH": "This area is linked to RoSH (risk of serious harm)",
-      "notLinkedToRoSH": "This area is not linked to RoSH (risk of serious harm)",
-      "linkedToReoffending": "This area is linked to risk of reoffending",
+      "linkedtoHarm": "This area is linked to RoSH (risk of serious harm)",
```

Comment:
change to 'Harm' is OK but can we keep the camel case on the rest of the field names?

Reply:
Done!